frontend/amer-fort-explorer/style.css

```
@@ -0,0 +1,985 @@
+  --amer-border: rgba(212, 163, 115, 0.4);
+  --amer-text: #2b2118;
+  --amer-muted: #6b5c50;
+  --amer-glass: rgba(255, 255, 255, 0.8);
+  --amer-shadow: 0 10px 30px rgba(138, 90, 50, 0.12);
+}
+
+.amer-main {
+  background-color: var(--amer-dark);
+  color: var(--amer-text);
+  min-height: 100vh;
+  font-family: 'Outfit', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)